tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
+  return {
+    get: (name) => headerData[name],
```

Comment:
Actually, it appears as though the `'Content-type'` header is special? In Chrome and Firefox it always returns and with multiple values it returns them comma-separated. I guess this means the MDN docs are incorrect.